pkg/compiler/transformers/k8s/workflow.go

```diff
@@ -171,6 +171,7 @@ func BuildFlyteWorkflow(wfClosure *core.CompiledWorkflowClosure, inputs *core.Li
 	if err != nil {
 		errs.Collect(errors.NewWorkflowBuildError(err))
 	}
+	obj.ObjectMeta.Labels[WorkflowIDLabel] = primarySpec.ID
```

Comment:
Is it really the full `ID`? or just the `Name` part of the ID?

Reply:
thanks for asking. It was project:domain:name, changed it to name. Added a simple test.